### DISCUSSION TITLE
Request for Swagger API to create a document in Wiki

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#4780] [General]
Guys, please tell me, we want to make a request via swagger for the API. To automatically create a page in a collection or Wiki section. To do this, you need to register "collectionId" and "parentDocumentId". The question is where and how can I find these "collectionId" and "parentDocumentId"?

Request via swagger via Wiki API to create a document.

Outline API
 0.1.0 
OAS3

request POST /documents.create  Create a document

description request

This method allows you to create or publish a new document. By default a document is set to the collection root. If you want to create a nested/child document, you should pass parentDocumentId to set the parent document.

Request body

{
  "title": "Welcome to Acme Inc",
  "text": "…",
  "collectionId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "parentDocumentId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "templateId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "template": true,
  "publish": true
}

**Comment by apoorv-mishra** [comment]
Reload your supposedly parent document once and look for the response of `documents.info` in devtools. The value of `id` there is what you need as `parentDocumentId` in your `documents.create` call. As for `collectionId`, copy as it from the same response. Hope this helps.
